addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/handler/NetatmoBridgeHandler.java

```diff
@@ -85,6 +90,18 @@ private String getApiScope() {
             stringBuilder.append("read_thermostat write_thermostat ");
         }
 
+        if (configuration.readWelcome) {
+            stringBuilder.append("read_camera ");
+        }
+
+        if (configuration.accessWelcome) {
+            stringBuilder.append("access_camera ");
+        }
+
+        if (configuration.writeWelcome) {
+            stringBuilder.append("write_camera ");
+        }
+
```

Comment:
There are 3 types of Access for the welcome api, just read without access to the camera pictures and streams, access with access to the urls for stream and write for writing back if someone or everybody left home.
For security reasons i think the user should decide how much access he wants to grant the api, but as you said its easier to give him full access.